typescript/infra/config/environments/mainnet3/warp/configGetters/exportConfigs.ts

```diff
@@ -0,0 +1,33 @@
+import { getArgs, withOutputFile } from '../../../../../scripts/agent-utils.js';
+import {
+  getEnvironmentConfig,
+  getHyperlaneCore,
+} from '../../../../../scripts/core-utils.js';
+import { getRegistry } from '../../../../registry.js';
+import { getWarpConfig, warpConfigGetterMap } from '../../../../warp.js';
+
+async function main() {
+  const { environment } = await withOutputFile(getArgs()).argv;
+  const { multiProvider } = await getHyperlaneCore(environment);
+  const envConfig = getEnvironmentConfig(environment);
+  const registry = getRegistry();
+
+  const warpIdsToCheck = Object.keys(warpConfigGetterMap);
+  for (const warpRouteId of warpIdsToCheck) {
+    console.log(`Generating Warp config for ${warpRouteId}`);
+
+    const warpConfig = await getWarpConfig(
+      multiProvider,
+      envConfig,
+      warpRouteId,
+    );
+
+    const configFileName = `${warpRouteId}.yaml`;
```

Comment:
Can we name it `${warpRouteId}-deploy-config.yaml` as per https://www.notion.so/hyperlanexyz/Warp-Speed-Part-1-Warp-Config-in-Registry-17b6d35200d680e8b465df2bd32cf42b?pvs=4#17b6d35200d6803da2ebf79b9e04af50

Reply:
fixed